backend/src/org/akvo/dash/import.sql

```diff
@@ -30,20 +30,20 @@ SELECT spec
 -- :name update-failed-job-execution :! :n
 -- :doc Update failed job execution
 UPDATE job_execution
-   SET finished_at=now(),
-       error_reason=:reason
- WHERE id=:id
+   SET log = :reason,
```

Comment:
As mentioned the `log` is just to hold statement(s) of the execution. If the status is `FAILED` you get the error from the log, if the status is `OK`, you could get the list of changes applied in the transformation. In the future will hold the execution log for any async process.